modules/data/webadmin/tmpl/add_edit_network.tmpl

```diff
@@ -21,15 +25,21 @@
 		<input type="hidden" name="user" value="<? VAR Username ?>" />
 		<input type="hidden" name="network" value="<? VAR Name ?>" />
 
+		<? IF !NameEdit ?>
+		<input type="hidden" name="name" value="<? VAR Name ?>" />
```

Comment:
This value already exists just above, why do you need another one?

Reply:
It does appear to be unnecessary, I'll remove it